docs/download.rst

```diff
@@ -26,7 +26,7 @@ We will show:
 **Setup**
 ---------
 
--  omero-downloader was installed by downloading and unzipping of the release artifact \ `https://github.com/ome/omero-downloader/releases/latest/ <https://github.com/ome/omero-downloader/releases/download/v0.2.1/OMERO.downloader-0.2.1-release.zip>`__
+-  omero-downloader was installed by downloading and unzipping of the release artifact \ `https://github.com/ome/omero-downloader/releases/latest/ <https://github.com/ome/omero-downloader/releases/download/v0.2.2/OMERO.downloader-0.2.2-release.zip>`__
```

Comment:
Thanks @mtbc for the PR. When reviewing, I am noticing that the link is saying ``latest`` in both the released version (see Resources in https://omero-guides.readthedocs.io/en/latest/download/docs/download.html ) and the version built with this PR (see Resources in https://omero-guides--2.org.readthedocs.build/projects/download/en/2/download.html).
But in the released doc, the ``latest`` link will download downloader 0.2.1, whereas the version of the doc built with this PR will download the downloader 0.2.2. 
This seems to me quite confusing. I would suggest to change the name of the link too, to say explicitly 0.2.2 or change the real link value also to ``latest``. What would you think @jburel ?